prow/plugins/label/label.go

```diff
@@ -119,6 +121,18 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 			continue
 		}
 
+		for _, label := range labels {
+			prefixes := strings.Split(*singleChoice, ",")
+			for _, prefix := range prefixes {
```

Comment:
This works but a better implementation is to make `prefixes` a map and loop on existing labels, remove outdated label if you found a match. Once found, you can break out the loop.

Reply:
move the `prefixesMap` construction loop to outside of the top-level for lop